app/src/main/java/com/peacecorps/pcsa/policies_glossary/GlossaryAdapter.java

```diff
@@ -0,0 +1,266 @@
+package com.peacecorps.pcsa.policies_glossary;
+
+import android.content.Context;
+import android.graphics.Typeface;
+import android.text.Html;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseExpandableListAdapter;
+import android.widget.ExpandableListView;
+import android.widget.TextView;
+
+import com.peacecorps.pcsa.R;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+/*
+ * Custom Adapter for Expandable List View in GlossaryFragment
+ * @author rohan
+ * @since 2016-07-30
+ */
+public class GlossaryAdapter extends BaseExpandableListAdapter {
+
+    private Context _context;
+    private List<String> _listDataHeader; // header titles
```

Comment:
There's another variable called listDataHeader in the same file. 
It's in a different scope altogether. So it would not give rise to conflicts but just for the sake of clarity, I did this.